client/server.crt

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
What's the purpose of this?

Reply:
Google Cast requires HTTPS to work, so this was a temporary certificate to perform some testing.

It's not meant to be kept in, but since this PR is essentially blocked until we figure out a solution for the queue, it's been kept in.